Geometry/HcalCommonData/src/CaloDDDSimulationConstants.cc

```diff
@@ -1,7 +1,7 @@
 #include "Geometry/HcalCommonData/interface/CaloDDDSimulationConstants.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-#define EDM_ML_DEBUG
+//#define EDM_ML_DEBUG
```

Comment:
I'm curious how this change made it to this PR.
@rappoccio please clarify

Reply:
Goood question. I rebased to 11_1_X and apparently this got picked up.

Reply:
@bsunanda @cvuosalo @fabiocos 
please clarify if this more recent commit/change is actually present in 11_0_X, but is not in 11_1_X.

https://github.com/cms-sw/cmssw/pull/28539#issuecomment-562222682 refers to #28506 as the master version for the change, but that PR did not have the change at least when it was merged (there was a similar HcalParametersFromDD.cc)

Reply:
It looks like one file was left out of #28506. Maybe it can be fixed in @bsunanda 's next PR.

Reply:
@cvuosalo @slava77 @rappoccio @bsunanda this is a useful fix that apparently was missing in master (and I overlooked this when integrating its backport) . Ok to avoid mixing it here, I'll fix it separately